src/module/actor/entity.js

```diff
@@ -134,7 +134,13 @@ export class OseActor extends Actor {
       details: game.i18n.format("OSE.roll.details.save", { save: label }),
     };
 
-    let skip = options?.event?.ctrlKey || options.fastForward;
+    const invertedCtrlBehavior = game.settings.get(game.system.id, "invertedCtrlBehavior");
```

Comment:
We're using this in a few places. We should extract it into its own method.

Reply:
Indeed. I made an effort for it in the latest commit, also to realize there are no tests written for monster rolls.

Reply:
re: no tests for, well, sheets in general -- I'd started writing tests for sheets, but we're also looking at a new sheet design (which we're looking at doing in Vueport). No sense doing work we're eventually tossing out :)

Reply:
Well that is certainly a good point :D